kadena/light-client/Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: benchmark bench-ci
+
+benchmark:
+	@read -p "Enter benchmark name: " bench; \
+	RUSTFLAGS="-C target-cpu=native -C opt-level=3" \
+	SHARD_SIZE=4194304 \
+	SHARD_BATCH_SIZE=0 \
+	SHARD_CHUNKING_MULTIPLIER=256 \
```

Comment:
Is this the optimal value of this parameter? For the 16x machine it was 32 and for the 32x machine - 64 - as far as I remember correctly.

Reply:
Currently running the benchmark on a `32xl` with `64`